DayCare/app/models/class_room.rb

```diff
@@ -0,0 +1,4 @@
+class ClassRoom < ActiveRecord::Base
+    has_many :children
```

Comment:
any methods for this class? just write each method's signature and comment inside explaining what it would do.